modin/pandas/test/dataframe/test_binary.py

```diff
@@ -197,3 +197,55 @@ def test_equals():
         df_equals(modin_df3, modin_df2)
 
     assert modin_df1.equals(modin_df2._query_compiler.to_pandas())
+
+
+@pytest.mark.parametrize("is_more_partitions_other", [True, False])
+@pytest.mark.parametrize(
+    "op_type", ["df_ser", "df_df", "ser_ser_same_name", "ser_ser_different_name"]
+)
+@pytest.mark.parametrize(
+    "is_aligned_idx", [True, False], ids=["idx_aligned", "idx_not_aligned"]
```

Comment:
`is_aligned_idx` -> `is_idx_aligned`, just for consistencies with `ids`.

Reply:
fixed